webpack.config.js

```diff
@@ -0,0 +1,24 @@
+var path = require('path');
+var webpack = require('webpack');
+var HtmlWebpackPlugin = require('html-webpack-plugin');
+
+let config = {
+	entry: './src/client/index.js',
+	output: {
+		path: path.join(__dirname, '/build'),
+		filename: 'index_bundle.js',
+		publicPath: '/'
+	},
+	module: {
+		loaders:[
+			{ test: /\.(js)$/, use: 'babel-loader' }
+		]
+	},
+	plugins: [
+		new HtmlWebpackPlugin({
+			template: './src/client/index.html'
+		})
```

Comment:
this plugin allows you to designate a base html template. the webpack plugin will then spit out a copy of this template with script tags added in the body for your index_bundle.js